src/main/java/ConsoleOut/ConsoleOutPreferences.java

```diff
@@ -0,0 +1,24 @@
+package ConsoleOut;
+
+import java.util.Scanner;
+
+public class ConsoleOutPreferences {
+    private int delay;
+    private String color;
+
+    public ConsoleOutPreferences() {
+        System.out.println("What color do you want the text to be?");
```

Comment:
Think what if I want to change colors not here, but from main menu of the game? It's hard-coded here, in some way. I want to give color and delay during creation of your writer. (Inversion of Control). If u would do this, this class is unnecessary (color and delay will be a field in ConsoleOutManagment).